.gitignore

```diff
@@ -45,4 +45,7 @@ docs/source/tools/*\.md
 docs/source/*/resources
 docs/source/tools/assets/
 .tfcache
-.env
+*.tfstate
+*.tfstate.backup
+*.tfvars
```

Comment:
any reason not to check in the vars? alternatively apply defaults to the var declarations so that this is usable without manual steps.

Reply:
felt like bad form to include the environment specializations in the checked in elements

Reply:
I will add a bootstrapper script that makes it work out of box without leaving tfvars droppings all over

Reply:
You can also set TF variables via [the environment](https://www.terraform.io/docs/configuration/variables.html#environment-variables) which would probably be a good alternative. Where sane defaults are set (location is a well known region, and suffix is something reasonable) then a user could set `TF_VAR_suffix` in shell to overwrite the default